ionhash/hasher.py

```diff
@@ -11,7 +11,8 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-"""Provides readers/writers that hash Ion values according to the Ion Hash Specification."""
+"""Adds an `ion_hash()` method to all simpleion value classes, and provides
```

Comment:
The addition of `ion_hash()` is done in `__init__.py`, but keeping this here is fine if the reasoning was that this module's doc is more discoverable by users.